src/System.Runtime.WindowsRuntime/src/System/IO/WindowsRuntimeStreamExtensions.cs

```diff
@@ -167,7 +167,7 @@ private static Stream AsStreamInternal(Object windowsRuntimeStream, Int32 buffer
             // There is already an adapter:
             if (adapterExists)
             {
-                Debug.Assert((adapter is BufferedStreamWrapper && ((BufferedStreamWrapper)adapter).UnderlyingStream is WinRtToNetFxStreamAdapter)
+                Debug.Assert((adapter is BufferedStream && ((BufferedStream)adapter).UnderlyingStream is WinRtToNetFxStreamAdapter)
```

Comment:
Given that you made the API netcoreapp1.1 specific I don't expect that this code can actually call it as it isn't compiled for netcoreapp1.1.  I suspect you are actually still using the internal version of this API in System.Private.CoreLib which I pointed out to @luqunl in his PR but it doesn't look like he fixed.